operations/admin.go

```diff
@@ -497,7 +497,7 @@ func adminBackup() cli.Command {
 			serviceConfigFlags(),
 			addDbSettingsFlags(
 				cli.StringFlag{
-					Name:  joinFlagNames(collectionNameFlag, "c"),
+					Name:  collectionNameFlag,
```

Comment:
This is unrelated: in looking for examples of Value for flags (which I've decided not to use) I tried `evergreen admin backup -h` which panics because -c is repeated, so I figured I would just fix it.